README.md

```diff
@@ -284,30 +284,36 @@ Obtain overlap-aware speaker embeddings from a microphone stream:
 ```python
 import rx.operators as ops
 import diart.operators as dops
-from diart.sources import MicrophoneAudioSource
+from diart.sources import MicrophoneAudioSource #, FileAudioSource
 from diart.blocks import SpeakerSegmentation, OverlapAwareSpeakerEmbedding
 
-segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation")
-embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding")
+segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation", use_hf_token="")
+embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding", use_hf_token="")
+
 mic = MicrophoneAudioSource()
+# To take input from file:
+# mic = FileAudioSource("<filename>", sample_rate=16000)
```

Comment:
```suggestion
source = MicrophoneAudioSource()
# To take input from file:
# source = FileAudioSource("<filename>", sample_rate=16000)
```

Changing the name to `source` because a file is not a mic